demo/Alsa cpp/main.cpp

```diff
@@ -0,0 +1,216 @@
+/*
+ * 
+ * Project: Porcupine Alsa
+ * Created Date: Tuesday, September 18th 2018, 7:41:55 pm
+ * Author: Greg
+ * -----
+ * Last Modified: Tue Sep 18 2018
+ * Modified By: Greg
+ * -----
+ * Copyright (c) 2018 Greg
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Could you please add the standard Apache 2 header? Since the repo is under Apache2 I am assuming you are OK releasing this code with same license?